File: Semester III/Programming in Python (4639304)/p5.py
x=int(input("Enter The Number .."))
y=int(input("Enter The Number .."))
z=int(input("Enter The Number .."))

if x>y and x>z:
	print("X is Max..",x)
elif y>x and y>z:
	print("Y is Max..",y)
else:
	print("Z is Max..",z)

if x<y and x<z:
	print("X is Min..",x)
elif y<x and y<z:
	print("Y is Min..",y)
else:
	print("Z is Min..",z)

File: Semester III/Programming in JAVA (4639302)/java Prog/p5.java
import java.util.*;
class p5
{
	public static void main(String[] args)
	{
		Scanner sc=new Scanner(System.in);
		
		System.out.println("Enter the No : ");
		
		int no=sc.nextInt();
		int no1=no;
		int rem,sum=0,temp=0;
		while(no!=0)
		{
			rem=no%10;
			sum=sum+(rem*rem*rem);
			temp=temp*10+rem;
			no=no/10;
		}
		System.out.println("Sum is : " + sum);
		
		if(sum==no1)
		{
			System.out.println("Number is Amstrong Number");
		}
		else
		{
			System.out.println("Number is Not Amstrong Number");
		}
	}
}